src/section.js

```diff
@@ -88,16 +88,9 @@ class Section {
 
 		this.load(_request).
 			then(function(contents){
-				var serializer;
-				var Serializer;
-
-				if (typeof XMLSerializer === "undefined") {
-					Serializer = require("xmldom").XMLSerializer;
-				} else {
-					Serializer = XMLSerializer;
```

Comment:
Hi @fchasen, 
We have used xmldom serializer as IE 11 xml serializer has issue converting xml to html string. It keeps tags self closed**(scripts tags and others)**, which doest render in IE at all. we can go with two approaches to fix it.

1.  Regex solution to fix self closing tags from the string.

2. Use  XMLHttpRequest.responseText directly.

Please suggest which approach to take.